circuits/cpp/barretenberg/cpp/src/barretenberg/honk/sumcheck/polynomials/barycentric_data.test.cpp

```diff
@@ -14,6 +14,19 @@ TYPED_TEST_SUITE(BarycentricDataTests, FieldTypes);
 
 #define BARYCENTIC_DATA_TESTS_TYPE_ALIASES using FF = TypeParam;
 
+/**
+ * @brief Ensure auxilliary arrays (e.g. big_domain) are computed at compile time if possible (i.e. if FF is literal)
```

Comment:
first, I noticed literal here and was confused what you are refering to

Reply:
changed language to native